src/cockpit/samples.py

```diff
@@ -169,10 +171,16 @@ def sample(self, samples: Dict[str, Any]):
                 if dev_name.startswith('nvme') and 'p' in dev_name:
                     continue
 
-                bytes_read += int(num_sectors_read) * 512
-                bytes_written += int(num_sectors_written) * 512
+                read_bytes = int(num_sectors_read) * 512
+                written_bytes = int(num_sectors_written) * 512
+
+                bytes_read += read_bytes
+                bytes_written += written_bytes
```

Comment:
I don't like how you're avoiding the naming conflict by basically changing the order and tense of the words.

I'd take the opportunity to change the name here to something more like `all_read_bytes` and `all_written_bytes`, and then use the `read_bytes` form for the per-instance value.

Reply:
Ah thanks for the suggestion! Yeah I didn't want to duplicate `(int(num_sectors_read) * 512)`.

I also wonder if this is wrong btw, we always multiple with 512, but are sectors always this size?

https://developer.ibm.com/tutorials/l-linux-on-4kb-sector-disks/ For example mentions 4kb sizes:

```
[jelle@t14s][~/projects/cockpit/main]%cat /sys/block/nvme0n1/queue/hw_sector_size
512
```

Now my ssd can't be formatted with 4 kb bock sizes, but still an interesting question.